app/jobs/alert_mailer_job.rb

```diff
@@ -32,7 +32,7 @@ def subscription
   end
 
   def subscription_id
-    arguments[3]
+    arguments.last[:args].first
```

Comment:
Moving to `MailDeliveryJob` means we have to mangle job arguments like this now as the method signature of `#perform` has changed:

```ruby
# DeliveryJob
def perform(mailer, mail_method, delivery_method, *args)
def perform(mailer, mail_method, delivery_method, args:, kwargs: nil, params: nil)
# MailDeliveryJob

```

https://github.com/rails/rails/blob/master/actionmailer/lib/action_mailer/delivery_job.rb#L23
https://github.com/rails/rails/blob/master/actionmailer/lib/action_mailer/mail_delivery_job.rb#L16